tests/components/modbus/conftest.py

```diff
@@ -11,7 +11,7 @@
     CONF_REGISTER_TYPE,
     CONF_REGISTERS,
     DEFAULT_HUB,
-    MODBUS_DOMAIN,
+    DOMAIN,
```

Comment:
```suggestion
    MODBUS_DOMAIN as DOMAIN,
```

Reply:
Yeah I just saw that one, I tried to solve another one by renaming DOMAIN.

This just goes to show, never to PRs at a late hour.